recipes/conversation_view/message_input/message_input.vue

```diff
@@ -611,11 +619,18 @@ export default {
      * @type {Boolean}
      */
     'cancel',
+
+    /**
+     * Fires when skin tone is selected from the emoji picker
+     *
+     * @event select-skin-tone
+     * @type {String}
+     */
+    'select-skin-tone',
```

Comment:
since emoji-picker already emits an event called "skin-tone" should we just use the same event name here?

Reply:
I went with that at first but for some reason it wasn't working when I was testing it in storybook. It was only when I named it something other than `skin-tone` that it'd properly emit the event. I didn't look into _why_ that was happening too deeply though, does that sound like just some weirdness on the storybook side of things?

Reply:
This might be only an issue with the naming, make sure to be listening for `onSkinTone` events on the components that need it, you have a prop called `skinTone` too so that might be confusing, maybe changing the property to `selectedSkinTone` 🤔

Reply:
I thought maybe it was a conflict between the prop and the event both being named skin-tone so I tried it out here: https://github.com/dialpad/dialtone-vue/pull/1313

Seems to work fine though, please confirm. Might have just been something you missed between the naming in the `stories.js` and the `.story.vue`

Reply:
Huh that's really weird, it's working completely fine now. I was banging my head on it for like an hour, I could've sworn everything was named properly 🤔 I'll merge this in, thanks Brad!